messages/en.json

```
@@ -372,6 +372,39 @@
+    "checkInManagement": "Check-in Management",
+    "checkInManagementDescription": "Display and print QR codes for each slot to manage participant entry and exit",
+    "slotQRCode": "Slot QR Code",
+    "slotNumber": "Slot Number",
+    "time": "Time",
+    "print": "Print",
+    "downloadQR": "Download QR Code",
+    "printAll": "Print All",
+    "refresh": "Refresh",
+    "checkInStatistics": "Check-in Statistics",
+    "totalBookings": "Total Bookings",
+    "checkedIn": "Checked In",
```

Biome (2.1.2)

[error] The key checkedIn was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)